### PR TITLE
devhub-1044  podcast and video players

### PR DESCRIPTION
**Pull request by harikakotipalli**
This PR replaces the place holders in podcast and video pages with podcast player and Youtube players.

https://devcenter-nf9z21p1d-harikakotipalli.vercel.app/videos/compass---the-gui-for-mongodb-in-10-mins----jumpstart

https://devcenter-nf9z21p1d-harikakotipalli.vercel.app/podcasts/ep--102-changing-the-game-with-mongodb

### CODE REVIEW

File: src/api-requests/get-all-videos.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const getAllVideosFromAPI = async (
 ): Promise<Video[]> => {
     const query = gql`
         query Videos {
-            videos @rest(type: "Video", path: "/videos") {
+            videos @rest(type: "Video", path: "/new-videos") {
```
Comment (harikakotipalli) [inline src/api-requests/get-all-videos.ts:15]: new collection @ajhenri populated data into

File: src/components/podcast-player/podcast-player.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import Script from 'next/script';
```
Comment (harikakotipalli) [inline src/components/podcast-player/podcast-player.tsx:1]: casted embedded player

File: src/components/card/card.tsx [hunk 3]
```diff
@@ -115,7 +116,9 @@ const Card: React.FunctionComponent<CardProps> = ({
                             />
                         )}
                         <TypographyScale variant="body3">
-                            {contentDate}
+                            {formatDateToDisplayDateFormat(
```
Comment (harikakotipalli) [inline src/components/card/card.tsx:119]: changing display format of date

File: src/service/get-all-content.ts [hunk 4]
```diff
@@ -58,7 +59,7 @@ export const mapPodcastsToContentItems = (
         if (p.thumbnailUrl) {
             item.image = { url: p.thumbnailUrl, alt: 'randomAlt' };
         }
-        item.podcastFileUrl = p.podcastFileUrl;
+        item.podcastFileUrl = p.casted_slug;
```
Comment (harikakotipalli) [inline src/service/get-all-content.ts:62]: needed for casted embedded player

**Review by ajhenri — APPROVED** [review]
LGTM

**Review by mcolella14 — APPROVED** [review]

**Pull request merged**